apps/api/src/coscrad-cli/__snapshots__/domain-restore.cli-command.e2e.spec.ts.snap

```diff
@@ -0,0 +1,2793 @@
+            "text": MultilingualText {
+              "items": [
+                MultilingualTextItem {
+                  "languageCode": "eng",
+                  "role": "original",
+                  "text": "by members of the opposite clan of the house chief",
+                },
+              ],
+            },
+          },
+        ],
+        "participants": [
+          TranscriptParticipant {
+            "initials": "E1",
+            "name": "Elder 1",
+          },
+          TranscriptParticipant {
+            "initials": "E2",
+            "name": "Elder 2",
+          },
+        ],
+      },
+      "type": "audioItem",
+    },
+  ],
+  "bibliographicReference": [
+    BookBibliographicReference {
+      "data": BookBibliographicReferenceData {
+        "abstract": "This is the abstract, NOT a general note!",
+        "creators": [
+          {
+            "name": "Alana Duvernay",
+            "type": "author",
+          },
+          {
+            "name": "James Smith",
+            "type": "author",
+          },
+        ],
+        "isbn": "978-1-895811-34-6",
+        "numberOfPages": 455,
+        "place": "Kennebunk Port, Maine",
+        "publisher": "Atlantic Publishing",
+        "title": "A Day in the Life",
+        "type": "book",
+        "url": "https://atlanticpublishing.com/authors/Duvernay/NewBook/BookInformation/",
+        "year": 1999,
+      },
+      "eventHistory": [],
+      "getCompositeIdentifier": [Function],
+      "id": "BIBLIOGRAPHICREFERENCE.9b1deb4d-3b7d-4bad-9bdd-2b0d7b110001",
+      "published": true,
+      "queryAccessControlList": AccessControlList {
+        "allowedGroupIds": [],
+        "allowedUserIds": [],
+      },
+      "type": "bibliographicReference",
+    },
+    JournalArticleBibliographicReference {
+      "data": JournalArticleBibliographicReferenceData {
+        "abstract": "An analysis of the Cariboo Chilcotin Justice inquiry.",
+        "creators": [
+          BibliographicReferenceCreator {
+            "name": "Sigurd Purcell",
+            "type": "author",
+          },
+        ],
+        "doi": "10.14288/bcs.v0i19.784",
+        "issn": "00052949",
+        "issueDate": "Spring 2013",
+        "publicationTitle": "Journal of History",
+        "title": "Report on the Cariboo Chilcotin Justice inquiry",
+        "type": "journalArticle",
+        "url": "https://search.proquest.com/docview/1682229477/abstract/7836BCEA06014582PQ/1",
+      },
+      "eventHistory": [],
+      "getCompositeIdentifier": [Function],
+      "id": "BIBLIOGRAPHICREFERENCE.9b1deb4d-3b7d-4bad-9bdd-2b0d7b110023",
+      "published": true,
+      "queryAccessControlList": AccessControlList {
+        "allowedGroupIds": [],
+        "allowedUserIds": [],
+      },
+      "type": "bibliographicReference",
+    },
+    CourtCaseBibliographicReference {
+      "data": CourtCaseBibliographicReferenceData {
+        "abstract": "William, June
+- Nemiah Valley
```

Comment:
Just making sure this is ok here?

Reply:
I'd rather not have real data in the test data. It's better to manually enter this into staging separately from our test data that we check in.

Reply:
I have anonymized the data.